release.md

```diff
@@ -0,0 +1,63 @@
+# Release Processes
+
+As per the [Versioning and release process of the libs artifacts](https://github.com/falcosecurity/libs/blob/master/proposals/20210524-versioning-and-release-of-the-libs-artifacts.md), this repository includes different groups of artifacts that needs to be versioned and released. Moreover, those artifacts follows different versioning schemas (see the [Driver SemVer](https://github.com/falcosecurity/libs/blob/master/proposals/20210818-driver-semver.md) and [Versioning schema](https://github.com/falcosecurity/libs/blob/master/proposals/20220203-versioning-schema-amendment.md)) proposals).
+
+We have two separate release processes which occur **independently** of each other: 
+- The **drivers release**
+- The **libs release** (ie. _libsinp_ and _libscap_)
+
+The drivers release process is mainly automated and happens outside this repository. The actual building and distribution system is implemented in our [test-infra](https://github.com/falcosecurity/test-infra) by the [Driverkit Build Grid](https://github.com/falcosecurity/test-infra/tree/master/driverkit), and drivers are published to https://download.falco.org/?prefix=driver/. The [drivers versioning](#Drivers-versioning) process happens in this repo. 
+
+>_Note that not all versioned releases will be built and distributed. Drivers distribution is indeed implemented and maintained only to satisfy Falco's needs._
+
+The libs release process is currently under development and limited to the versioning process only. The [libs versioning](#Libs-versioning) process happens in this repo.
+
+
+## Release procedure
+
+Regardless if it is a driver or a libs release when initiating a new release, we do the following process:
+
+1. We decide together (usually in the #falco channel in [slack](https://kubernetes.slack.com/messages/falco)) if the source code is a good shape and if it's the case to be released
+2. We double-check if the versioning rules have been respected (see sections below), then we pick the next version number to tag (i.e., a _git tag_)
+3. A person with repository rights creates a [new release using the GitHub UI](https://github.com/falcosecurity/libs/releases/new) (a git tag will be automatically created)
+
+> _At the time of writing, no other steps are needed since the whole release process is still in development. This document will be updated once the definitive process is fully implemented._
+
+
+## Drivers versioning
+
+The *driver version number* represents the build version of kernel-space drivers (i.e., the kernel module, the eBPF probe, and possibly any other kernel-space drivers).
```

Comment:
Maybe we can use `artifact` here instead of `drivers` to make it even more generic?
```suggestion
The *driver version number* represents the build version of kernel-space drivers (i.e., the kernel module, the eBPF probe, and possibly any other kernel-space artifact).
```